.github/workflows/macstadium-e2e.yml

```diff
@@ -41,19 +41,22 @@ jobs:
 
       - name: Check for frozen lockfile
         run: yarn check-lockfile
-        
+
       - name: Audit CI
         run:  yarn audit-ci --moderate --config audit-ci.json
 
       - name: Lint
         run: yarn lint:ci
-        
+
+      - name: Jest
```

Comment:
I'd rename this as "Unit tests" since e2e also uses Jest as a runner and it can be confusing

Reply:
Good call, done!